tests/core/test_nlg.py

```diff
@@ -56,6 +56,9 @@ async def generate(request):
     return app
 
 
+# TODO: Not sure if these ones can be replaced
+# pytest-sanic adds a fixture which is what is being used here
+# Not certain this won't change behaviour
```

Comment:
Not sure there's as much value changing the tests, seems like using the specific test fixtures may give us less headaches in the long run than using asyncio.run. What do you think?

Reply:
yes, agree 👍🏻